plugins/modules/proxmox_snap.py

```diff
@@ -74,6 +74,12 @@
       - Name of the snapshot that has to be created/deleted/restored.
     default: 'ansible_snap'
     type: str
+  trim:
+    description:
+      - Trim snapshots if there are more than C(trim) snapshots.
+      - If C(trim) is set to 0, all snapshots will be kept.
```

Comment:
```suggestion
      - Trim snapshots if there are more than O(trim) snapshots.
      - If O(trim) is set to V(0), all snapshots will be kept.
```

Reply:
I would call that param `retention` as it is a widespread name for that type of operation.

Reply:
That is a great suggestion :bow:. It is obviously the right name for the feature.